packages/itwinui-variables/src/themes/light-hc.scss

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@use 'sass:map';
+@use '../utils/hex' as *;
+
+$base-colors: (
+  // Background colors - default
+  background-1: hsl(200 12% 100%),
+  background-2: hsl(200 12% 98%),
+  background-3: hsl(200 12% 94%),
+  background-4: hsl(200 12% 88%),
+  background-5: hsl(200 12% 80%),
+  background-1-hover: hsl(0 0% 95%),
+  //
+  // Background colors - statuses subtle
+  background-subtle-primary: hsl(202 73% 94%),
+  background-subtle-positive: hsl(94 41% 93%),
+  background-subtle-warning: hsl(34 85% 95%),
+  background-subtle-negative: hsl(0 67% 94%),
```

Comment:
Here are the values for light high contrast subtle colors:
```scss
  background-subtle-primary: hsl(200 25% 95%),
  background-subtle-positive: hsl(135 15% 95%),
  background-subtle-warning: hsl(48 20% 95%),
  background-subtle-negative: hsl(0 36% 96%),
```

Reply:
Added